core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiService.scala

```diff
@@ -190,6 +190,16 @@ trait EntityApiService extends UserInfoDirectives {
               }
             }
           }
+        } ~
+        path("workspaces" / Segment / Segment / "entityTypes" / Segment / "renameAttribute") { (workspaceNamespace, workspaceName, entityType) =>
+          post {
+            entity(as[AttributeRename]) { attributeRenameRequest =>
+              complete {
+                entityServiceConstructor(userInfo).renameAttribute(WorkspaceName(workspaceNamespace, workspaceName),
+                  entityType, attributeRenameRequest).map(_ => StatusCodes.NoContent)
+              }
+            }
+          }
```

Comment:
I'm gonna sleep on this API signature. IMHO it's still inconsistent with rename-entity-type, so maybe we should just move that other API …

Reply:
I think what you're saying is that the "entityTypes"/ <entityType> part of the path is inconsistent with the "entities" / <entityType> part of the other API. I'm strongly in favor of the "entityTypes" approach since we're not overloading the "entities" / <entityType> / <entityName> path, and thus not introducing potential edge cases about what an entity name can be. I'll update the other API signature to match-

As far as POST over PATCH, passing both old & new attributes in the request body, the "verb" at the end of the path- I believe both signatures are consistent in all of those regards.